### PR TITLE
Corrections in tests

### PR DESCRIPTION
**Pull request by mrajashree**
Added conditions to check if the template objects and urls array are null.

### CODE REVIEW

File: integration/core/test_api.py [hunk 1]
```diff
@@ -196,21 +196,22 @@ def test_template_upgrade_version_links(client):
...
-            resp = response.json()
-            assert resp['bindings'] is not None
-            for i in range(len(resp['bindings'])):
-                services = resp['bindings']['services']
-                for service in services:
-                    service_map = resp['bindings']['services'][service]
-                    assert service_map['labels'] is not None
-                    assert service_map['ports'] is not None
+    templates = client.list_template(catalogId='qa-catalog')
+    if len(templates) > 0:
```
Comment (joshwget) [inline integration/core/test_api.py]: Why was the assertion taken out here? If length is 0 we might get a false positive on this test.

**Comment by mrajashree** [comment]
@ibuildthecloud does this 'wait for' logic look sufficient?

**Comment by ibuildthecloud** [comment]
@mrajashree You can probably just add that once to the `client` fixture and then you don't need it ever time you write a test.

**Comment by ibuildthecloud** [comment]
@mrajashree in `def client()`, where the client is constructed, can you just add this wait in that one place.

**Comment by mrajashree** [comment]
@ibuildthecloud Added the code in def client() fixture

**Pull request merged**